Makefile

```
@@ -662,6 +667,27 @@ _clean-azure-conditional:
+	else \
+		./scripts/cleanup-aws-resources.sh --region "$(REGION)"; \
+	fi
+
+# Internal target: force delete all AWS resources without prompting
+.PHONY: _clean-aws-force
+_clean-aws-force:
+	@./scripts/cleanup-aws-resources.sh --region "$(REGION)" --force 2>/dev/null || true
+
+# Internal target: conditionally clean AWS resources (only for ROSA)
+.PHONY: _clean-aws-conditional
+_clean-aws-conditional:
```

checkmake (0.3.2)

[warning] Required target "all" is missing from the Makefile.

(minphony)